Scripts/run_ss_on_exp_sim/scripts/get_sum_stats.py

```diff
@@ -0,0 +1,220 @@
+#import sys
+#sys.path.append('../../../')
+import os
```

Comment:
You have multiple imports of os here. It would be cleaner to import just os and update function calls accordingly.